logger.js

```diff
@@ -0,0 +1,11 @@
+const loggerHelper = (typeOfLog, message, arg = '') => {
+    console[typeOfLog](message, arg)
+};
+
+
+module.exports = {
+    log: (message, arg) => loggerHelper('log', message, arg),
```

Comment:
Approved but better to allow passing as many args as possible
console.log(1,2,3,4);
```
const loggerHelper = (typeOfLog, ...args) => {
    console[typeOfLog](...args)
};
log: (...args) => loggerHelper('log', ...args)
````

Reply:
Fixed